experiments/train_test_overlap/eval_datasets_overlap.py

```diff
@@ -0,0 +1,369 @@
+from marin.execution.executor import ExecutorStep, executor_main, this_output_path, versioned
+from operations.download.huggingface.download import DownloadConfig
+from operations.download.huggingface.download_hf import download_hf
+from operations.raw2json.huggingface.qa.raw2json import DatasetConversionConfig, OutputFormatOptions, raw2json
+
+"""
+This script downloads HF datasets and converts them to dolma "text" format for decontamination.
+We use the decontamination format to identify potential overlaps between pre-training data and evaluation data.
+
+The script follows the pattern from eval_datasets.py but focuses only on the decontamination conversion.
+"""
+
+############################################################
+# Download datasets
+############################################################
+
+# mmlu raw
+mmlu_raw = ExecutorStep(
```

Comment:
We already have these in `experiments/eval_datasets.py`; try not to copy if you can help it.